modules/gapi/include/opencv2/gapi/infer/parsers.hpp

```diff
@@ -52,7 +52,7 @@ namespace parsers {
 /** @brief Parses output of SSD network.
 
 Extracts detection information (box, confidence, label) from SSD output and
-filters it by given confidence and label.
+filters it by given confidence and label. FBL - filter by label.
```

Comment:
According to the `OOB` it can be just `BL`

Reply:
Added specialization instead.